packages/web-components/fast-components/.storybook/preview.js

```diff
@@ -1,6 +1,4 @@
-import * as FAST from "../src/index-rollup";
-
-FAST;
```

Comment:
Yep, it creates an unnecessary import when what we really want here are the side effects from that import.

Since the `fast-components` package manifest has `"sideEffects": false`, any Webpack-based system (like Storybook) will assume that unused imports can be tree-shaken away. This can be a problem since all of our exported components are essentially side effects: they get added to the `CustomElementRegistry` when initialized.